bin/development-server.js

```diff
@@ -26,10 +38,22 @@ app.use(webpackDevMiddleware(compiler, {
   }
 }));
 
+app.use(webpackDevMiddleware(babelCompiler, {
+  publicPath: "/babel",
+  noInfo: true,
+  stats: {
+    colors: true
+  }
+}));
+
 app.get("/", function(req, res) {
   res.sendFile(path.join(__dirname, "../index.html"));
 });
 
+app.get("/babel", function(req, res) {
+  res.sendFile(path.join(__dirname, "../babel.html"));
+});
+
```

Comment:
The tests visit `8000/babel` so that headlessly the es6 code is traspiled to es5. Hopefully this can go away soon.